src/CodeGeneration.Roslyn.Tasks/build/CodeGeneration.Roslyn.BuildTime.props

```diff
@@ -1,14 +1,7 @@
-<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <CodeGenerationRoslynToolsRootPath Condition=" '$(CodeGenerationRoslynToolsRootPath)' == '' ">$(MSBuildThisFileDirectory)</CodeGenerationRoslynToolsRootPath>
-    <CodeGenerationRoslynToolsSubPath Condition=" '$(MSBuildRuntimeType)' == 'Core' ">netstandard1.5\</CodeGenerationRoslynToolsSubPath>
-    <CodeGenerationRoslynToolsSubPath Condition=" '$(MSBuildRuntimeType)' != 'Core' ">net45\</CodeGenerationRoslynToolsSubPath>
-  </PropertyGroup>
-
-  <ItemDefinitionGroup>
-    <Compile>
-      <Generator>MSBuild:GenerateCodeFromAttributes</Generator>
-    </Compile>
-  </ItemDefinitionGroup>
+  <ItemGroup>
+    <!--This item's Version metadata is updated to match the package's one in .csproj-->
```

Comment:
Can you elaborate on this comment to specify that the substitution is done at build-time by the `SetPackageVersionInProps` target?

Reply:
Will do.